src/argilla/training/transformers.py

```diff
@@ -302,7 +299,6 @@ def compute_metrics(p):
             raise NotImplementedError("Text2Text is not implemented.")
         return func
 
-    # @require_version("setfit", "0.6")
     def train(self, output_dir: str = None):
```

Comment:
```suggestion
    def train(self, output_dir: str):
```
If it's required (see the ValueError below), then can't we just require it in the signature like this?